test-cases/scale/longevity-elasticity-many-small-tables.yaml

```diff
@@ -0,0 +1,40 @@
+#  This is a test case for having 90% disk utilization with a lot of small tables.
```

Comment:
Extra space at the start of the sentence

Reply:
fixed